doc/releases/pacific.rst

```diff
@@ -1547,7 +1547,7 @@ Post-upgrade
    the `crush-compat` :ref:`balancer <balancer>` mode added back in Luminous.
 
 #. If you did not already do so when upgrading from Mimic, we
-   recommened you enable the new :ref:`v2 network protocol <msgr2>`,
+   recommended you enable the new :ref:`v2 network protocol <msgr2>`,
```

Comment:
This fix seems okay to me, but not the ones in commit messages above.  Arguably those should have been fixed before merging, but in the words of Rainbow and Carole King, it's too late.